src/CFamily.UnitTests/CMake/CompilationDatabaseLocatorTests.cs

```diff
@@ -27,8 +27,6 @@
 using SonarLint.VisualStudio.CFamily.CMake;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.CFamily;
-using SonarLint.VisualStudio.Infrastructure.VS;
```

Comment:
I originally removed all unnused references and tried to put back the ones that are not relevant to this pr. However, I definetely missed a few but it should not really matter.